src/projection_client.rs

```diff
@@ -0,0 +1,616 @@
+use crate::event_store::client::projections;
+use crate::event_store::client::shared::Empty;
+use crate::grpc::{ClientSettings, GrpcClient};
+use crate::options::projections::{CreateProjectionOptions, UpdateProjectionOptions, DeleteProjectionOptions, ResetProjectionOptions, GetStateProjectionOptions, GetResultProjectionOptions};
+use crate::Credentials;
+use futures::stream::BoxStream;
+use serde::de::DeserializeOwned;
+
+#[allow(dead_code)]
+#[derive(Clone, Debug)]
+pub(crate) enum StatsFor {
+    Name(String),
+    AllProjections,
+    AllContinuous,
```

Comment:
**CONCERN:** Aren't now all projection continuous?

Reply:
This is only used internally, the gRPC files don't reflect that part of the RFC yet.

Reply:
Ah, ok 👍